src/kibana/components/vislib/visualizations/_point_series_chart.js

```diff
@@ -100,22 +102,19 @@ define(function (require) {
 
       function callPlay(event) {
         var boundData = event.target.__data__;
+        var offsetX = $chartEl.offset().left;
         var wholeBucket = boundData && boundData.x != null;
 
+        // the min and max that the endzones start in
         var min = leftEndzone.w;
         var max = rightEndzone.x;
 
         // bounds of the cursor to consider
-        var xLeft = event.offsetX;
-        var xRight = event.offsetX;
-        if (wholeBucket) {
-          xLeft = xScale(boundData.x);
```

Comment:
This is what I'm talking about: 
![2015-05-26 14_00_03](https://cloud.githubusercontent.com/assets/1329312/7823323/8c623da8-03af-11e5-9b67-5fcada8b5d88.gif)